Neos.Cache/Classes/Backend/FileBackend.php

```diff
@@ -245,11 +245,15 @@ public function findIdentifiersByTag(string $searchedTag): array
      * specified tags.
      *
      * @param string[] $tags The tags to search for
-     * @return string[] An array with identifiers of all matching entries. An empty array if no entries matched
+     * @return string[] An array with identifiers of all matching entries. An empty array if no entries matched or no tags were provided
      * @api
      */
     public function findIdentifiersByTags(array $tags): array
     {
+        if (empty($tags)) {
```

Comment:
```suggestion
        if ($tags === []) {
```

Reply:
hmm true but in this case its actually fully equivalent ^^ so im fine with merging it either way.

Reply:
I chose `empty` solely for readability [because `empty($tags)` gets parsed as `!$tags` anyway](https://github.com/php/php-src/blob/98b207ed8aa815afbda777dd338ff1847c7e9cff/Zend/zend_compile.c#L9587). So performance wise it should make no difference

Reply:
I think there is always leeway for some personal preference, I also like to use empty because it's just an easy to parse "keyword" in a condition.

Reply:
Linking to the actual line in `php-src` 😂 pro gamer move 💯